frontend/src/views/web/contact-us.html

```diff
@@ -0,0 +1,60 @@
+<div class="white-bg view-container">
+	<main-header></main-header>
+    
+	<div class="grad-container">
+		<div class="row">
+			<div class="col s12 m8 l8">
+				<br><br>
+				<h4 align="center" class="text-dark-black"><strong>Contact Us</strong></h4>
```

Comment:
@Ram81  use css property `text-align:center` rather than using html attribute for the alignment